src/common/contracts/ao-process.ts

```diff
@@ -23,6 +24,143 @@ import { version } from '../../version.js';
 import { WriteInteractionError } from '../error.js';
 import { ILogger, Logger } from '../logger.js';
 
+export const STUB_PROCESS_ID = 'process-id-'.padEnd(43, '1');
+export const STUB_ADDRESS = 'arweave-address-'.padEnd(43, '1');
+export const STUB_ETH_ADDRESS = '0xFCAd0B19bB29D4674531d6f115237E16AfCE377c';
+export const STUB_ANT_REGISTRY_ID = 'ant-registry-'.padEnd(43, '1');
+
+export const AO_LOADER_HANDLER_ENV = {
+  Process: {
+    Id: STUB_ADDRESS,
+    Owner: STUB_ADDRESS,
+    Tags: [
+      { name: 'Authority', value: 'XXXXXX' },
+      { name: 'ANT-Registry-Id', value: STUB_ANT_REGISTRY_ID },
+    ],
+  },
+  Module: {
+    Id: ''.padEnd(43, '1'),
+    Tags: [{ name: 'Authority', value: 'YYYYYY' }],
+  },
+};
+
+export const AO_LOADER_OPTIONS = {
+  format: 'wasm32-unknown-emscripten-metering',
+  inputEncoding: 'JSON-1',
+  outputEncoding: 'JSON-1',
+  memoryLimit: '1073741824', // 1 GiB in bytes
+  computeLimit: (9e12).toString(),
+  extensions: [],
+};
+
+export type HandleFunction = Awaited<ReturnType<typeof AoLoader>>;
+
+export class LocalAO implements Partial<AoClient> {
```

Comment:
put all this in tests, don't want to export it from the SDK nor the related dependencies

Reply:
Oh - I was actually thinking this might be useful for testing on the arns portal. But can figure that out later, will move.

Reply:
we can harden the pattern/class here and then potentially move it to a separate testing library